fineract-provider/src/main/resources/db/changelog/tenant/parts/20243008011009122_SU_384_add_report_loan_archive_history.xml

```diff
@@ -0,0 +1,133 @@
+    segundo_apellido AS "Segundo Apellido",
+    estado_cliente AS "Estado Cliente",
+    numero_obligacion AS "Numero Cuenta Cuota",
+    nit_empresa AS "Nit Empresa",
+    telefono_sac_1 AS "Telefono",
+    celular_sac_1 AS "Mobile No",
+    telefono_sac_2 AS "Telefono Sac 2",
+    celular_sac_2 AS "Celular Sac 2",
+    email_sac AS "Email",
+    direccion_sac AS "Direccion",
+    barrio_sac AS "Barrio",
+    ciudad_sac AS "Ciudad",
+    departamento AS "Departamento",
+    tipo_credito AS "Categoria punto de venta de origen del credito",
+    fecha_vencimiento AS "Vencimiento cuota",
+    dias_mora AS "Dias de mora de la cuenta",
+    valor_cuota AS "Cuota",
+    capital AS "Capital",
+    aval AS "Aval",
+    intereses AS "Intereses",
+    intereses_de_mora AS "Intereses de mora",
+    iva_interes_de_mora AS "IVA de mora",
+    seguro AS "Seguro obligatorio cuota",
+    seguros_voluntarios AS "Seguros voluntarios cuota",
+    punto_de_venta AS "Punto de venta de origen del credito",
+    tipo_documento AS "Tipo de documento",
+    empresa AS "Empresa",
+    marca AS "Nombre Aliado",
+    razon_social AS "Nombre completo del cliente",
+    ciudad_punto_credito AS "Ciudad punto de venta",
+    estado_cuota AS "Estado de la cuenta",
+    nombre_familiar_1 AS "Referencia del cliente",
+    parentesco_familiar_1 AS "Parentesco referencia",
+    celular_familiar_1 AS "Celular referencia",
+    nombre_familiar_2 AS "Nombre familiar",
+    parentesco_familiar_2 AS "Parentesco familiar",
+    celular_familiar_2 AS "Celular familiar",
+    fecha_financiacion AS "Fecha de desembolso",
+    fecha_nacimiento AS "Fecha nacimiento cliente",
+    genero AS "Genero cliente",
+    empresa_labora AS "Nombre empresa donde labora",
+    ingresos AS "Media de ingresos",
+    periodicidad AS "Mensual",
+    antiguedad_cliente AS "Fecha de creacion del cliente",
+    empresa_reporta AS "Intercredito",
+    abono AS "Abonos cuota",
+    condonaciones AS "Condonaciones cuota",
+    actividad_laboral AS "Actividad laboral del cliente",
+    cre_estado AS "Estado cliente",
+    cre_saldo AS "Saldo total del credito",
+    cuo_saldo AS "Saldo total cuota",
+    cuo_estado AS "Cuo Estado de la cuenta",
+    estado_civil AS "Estado civil del cliente",
+    numero_de_reprogramaciones AS "Numero de reprogramaciones" FROM m_archive_loan_history
+                ]]>
+            </column>
+            <column name="report_type" value="Table"/>
+            <column name="report_category" value="Loan"/>
+            <column name="description" value="Reporte de Archivo de cartera"/>
+            <column name="core_report" value="false"/>
+            <column name="use_report" value="true"/>
+        </insert>
+        <insert tableName="stretchy_report_parameter">
+            <column name="report_id"
+                    valueComputed="(select id from stretchy_report where report_name = 'Daily Portfolio')"/>
+            <column name="parameter_id"
```

Comment:
@tabrez-fiter Why adding report parameters when there is no where clause in the report query?